docs/glossary/voting.md

```diff
@@ -0,0 +1,17 @@
+## Voting
+
+Core to the [Steem blockchain](/docs/glossary/steem-blockchain.md) logic is the action of voting. There are 2 types of votes on the network:
+
+- Vote for [witnesses](/docs/glossary/witness.md) to be block validators
+- Vote for content
+
+The fundamental unit of account on the Steem platform is [STEEM](/docs/glossary/steem.md), a cryptocurrency token. Steem operates on the basis of one-STEEM, one-vote. Under this model, individuals who have contributed the most to the
+platform, as measured by the amount of [vested tokens](/docs/glossary/steem-power.md) in their account, have the most influence over how contributions are scored and which witness becomes approved by the community (the top 21 witnesses).
+
+## Token Rewards
+
+Token rewards are distributed by the amount of [VESTS](/docs/glossary/vests.md) backing the votes on a given article. How much is distributed is dictated by the token's [reward curve](/docs/glossary/reward-curve.md), [reward pool](/docs/glossary/reward-pool.md) and how many people are competing for it as well as each users [MANA](/docs/glossary/mana.md). 
+
+
```

Comment:
Why three empty lines at the end of file @RenaudGagne

Reply:
Thank you @ms10398, I went through everything and reformated:
- https://github.com/steemdocs/steemdocs/commit/7a421258122ffce22c8ed645a6f97d11ac7d60b8